### ISSUE TITLE
Simple error free method to download and install Cheat Engine

### ISSUE THREAD

**Issue by fade2gray** [cheat-engine/cheat-engine#1954]
See #2099 also.
Installing CE is a multistep process.

1. Download the installer from the CE website.
2. Run the installer.
3. The installer connects to a second web address (cloudfront.net) to download more files, after which the installation process continues.

Situations that might cause problems installing Cheat Engine.

- You have disconnected from the internet before step 2.
- You are using a VPN, proxy or a PiHole.
- Your ISP is blocking your connection to the second web address.
- An anticheat has been installed by a game you have played.

Just add an exclusion for your browser's download folder in `Windows Security > Virus & threat protection`,
and disable `Windows Reputation-based protection`. You may also need to disable any 3rd party firewall and/or antivirus software.

NOTE: You may also need to add an exclusion for your temp folder `C:\Users\<username>\AppData\Local\Temp`.

Video best viewed full-screen.

https://user-images.githubusercontent.com/1332009/149831217-ebe5cefb-1c7e-487f-9239-90c94f8cf930.mp4

(Updated video. Note, the current 7.3 installer actually installs CE 7.4)

**Comment by buchon1** [comment]
file:///C:/Users/chiki/Downloads/WhatsApp%20Image%202022-02-10%20at%2012.58.54%20PM.jpeg


This appears to me when I open cheat engine and reinstall cheat engine I also deactivated the antivirus and allowed an exception in Windows security for the downloads folder. And it still won't let me

**Comment by fade2gray** [comment]
@buchon1 

Drag and drop the image into the comments section. 

Arrastra y suelta la imagen en la sección de comentarios.

**Comment by AaronCatolico** [comment]
This is bullshit. Even far more advanced programs that have better patching & software manipulation capabilities, do NOT have ANY viruses or malware warnings. There's NO good reason for any software like this to have well over 29+ virus, malware, & adware warnings (according to VirusTotal's website)!

**Comment by fhwo13** [comment]
After I excluded and installed CE in the antivirus software, I found that my chrome was automatically prompted to install an extension called Online Security, which is very rogue

**Comment by AaronCatolico** [comment]
> Just add an exclusion for your browser's download folder in `Windows Security > Virus & threat protection`,
> and disable `Windows Reputation-based protection`. You may also need to disable any 3rd party firewall and/or antivirus software.
> 
> NOTE: You may also need to add an exclusion for your temp folder `C:\Users\<username>\AppData\Local\Temp`.
> 
> Video best viewed full-screen.
> 
> https://user-images.githubusercontent.com/1332009/149831217-ebe5cefb-1c7e-487f-9239-90c94f8cf930.mp4
> 
> (Updated video. Note, the current 7.3 installer actually installs CE 7.4)
> 
> 
> 
**
Basically just disable all forms of security so that you can install all of the Cheat Engine viruses, aware, malware, etc., etc..**

**Comment by fade2gray** [comment]
@AaronCatolico CE has no viruses, and you seem totally oblivious to the fact that you can skip the installation of all optional software offered - point in case, 3 minutes and 45 seconds into the video. Man up and own your shortcomings.

**Comment by AaronCatolico** [comment]
> @AaronCatolico CE has no viruses, and you seem totally oblivious to the fact that you can skip the installation of all optional software offered - point in case, 3 minutes and 45 seconds into the video. Man up and own your shortcomings. Oh, and you claim to be a "Software Developer & Web Developer"???

Incorrect, sir. Last time I tested it out, VirusTotal . com clearly said that it was FULL of viruses, adware, malware, etc., and the FAKE checkbox options still dont prevent the viruses, malware, adware, etc., from being installed on your machine. Even after the FAKE installation, the program never runs because the antivirus from Windows quickly shuts it down. There's still no acceptable excuse for the program to have this many detections in it. Being a programmer has no relevance to this discussion.

**Comment by AaronCatolico** [comment]
> @AaronCatolico Been seeing this bs topic over and over lately, maybe you should upload the cheatengine executable and not the whole installer executable. Most installers that have optional installs are flagged by malware and virus scanners. Its called a false positive. I have been using and modding CE for years. It dont have viruses. Its flagged cause its a hacking tool, a disassembler, kernel mod, reverse engineering tool, it has certain things in it that set off virus scans and malware detectors. Upload the source zip and see what it says ahem it detects cheatengine
> ![254A412D-7CBF-4B94-A8D6-ABEBD4705DE8](https://user-images.githubusercontent.com/211848/226075783-82bb4d19-91ca-40b4-b54d-d6132169f9db.png)
> 

That's still NOT acceptable. I've used plenty of disassemblers, hacking tools, etc., and none of them had over 20-30+ detections like this program has. I've done reversing with x64dbg, Ollydgb, etc., and none of them have any of this nonsense.

**Comment by rcg4u** [comment]
You are kidding right. Those are detecting cheatengine because someone like you cried to a malware/antivirus company Wahhh I installed a hacking tool and i think it infected me with nothing and they decided to add a part of the code as being detected. Not acceptable? lol that second image the zip only showing cheatengine being detected. If you are this paranoid about cheatengine wait till you find out NSA has a backdoor for every windows install. Its built in by microsoft
